src/Okta.Idx.Sdk.E2ETests/Helpers/TestContext.cs

```diff
@@ -30,7 +30,7 @@ public TestContext(ITestConfiguration configuration, IA18nClient a18nClient, IOk
             _a18nClient = a18nClient;
             _configuration = configuration;
 
-            _a18nProfile = _a18nClient.GetProfileAsync().Result;
+            _a18nProfile = Task.Run(() => _a18nClient.GetProfileAsync()).Result;
```

Comment:
I think this can be simplified to `_a18nClient.GetProfileAsync().Result;`; that is unless `GetProfileAsync` is not actually an async method.

Reply:
It can't unfortunately, I had a deadlock with this simplified call.